content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx

```diff
@@ -74,16 +74,19 @@ export class Hero extends React.PureComponent {
               <div className="img" style={{backgroundImage: `url(${heroRec.image_src})`}} />
             </div>
             <div className="meta">
-              <div className="header-and-excerpt">
-                <header>{heroRec.title}</header>
-                <p className="excerpt">{heroRec.excerpt}</p>
-              </div>
+              <header>{heroRec.title}</header>
```

Comment:
I'll fix this loss of `<div className="header-and-excerpt">` in https://github.com/mozilla/activity-stream/pull/4798